src/app/connect-wallet-alchemy/connect-wallet-alchemy.component.html

```diff
@@ -0,0 +1,83 @@
+<div class="h-screen bg-gradient-to-b from-green-100 to-brand-turquoise">
```

Comment:
![image](https://github.com/Venture-Miner/w3fs/assets/111710515/750f82ab-e45a-457c-a054-f26852ddaca0)

The logo is barely visible. It would be nice if there was some contrast